lib/web_ui/dev/driver_version.yaml

```diff
@@ -1,4 +1,10 @@
-
+## Version to be picked by this PR. The LUCI will have multiple cipd packages
+## insalled in the recipe at the same time.
+## This mapping will be used to switch between them.
+preferred_version:
```

Comment:
Do we need the `preferred_version` key? Is it going to contain more key/value pairs in the future? I think `preferred_chrome_version: 84` should be sufficient.

I'm also confused by the term "preferred". What does it mean?

Reply:
There are two versions recipe uploads at the same time. This `sha` prefers one version. It has to be like that otherwise some PRs who has the previous version in this file will start failing. To sum up, updating the Chrome version is a 4 step operation.

Steps to add a new version:
- Create a CIPD package
- Add the line to recipe to download this cipd package (keep the previous version as well). Tree will stay green this way
- Send a change PR and update the preferred version.
- After 1 week, hopefully all PRs will already be synced to HEAD. Remove the previous Chrome version from the recipe. There is currently no way to delete a CIPD package.

Reply:
Do we need the preferred_version key? -> Yes we need one for new edge. 

Firefox can work with multiple versions so it does not need this.

Reply:
Does the following not work?

```yaml
chrome_version: 84
edge_version: 85
...
```

Reply:
Why use it instead of what's in the PR? 
```
preferred_version:
   chrome: 84
   edge: 85
```
I don't see any practical value of changing it.

Reply:
"preferred" typically means that you will choose the specified version if it's available, but you will fallback to another version otherwise ("I prefer a blackberry croissant, but a plain croissant is also fine"). Is that the semantic we're going for? What happens if the specified version is not available?

Reply:
done. readme and documentation is updated.